pod-security/mutate/set-run-as-non-root/set-run-as-non-root.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: set-run-as-non-root
+  annotations:
+    policies.kyverno.io/title: Set runAsNonRoot
+    policies.kyverno.io/category: Pod Security Standards (Mutate)
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kyverno-version: 1.6.0
```

Comment:
This label reflects the version of Kyverno on which the policy was tested and confirmed working. Should be the 1.12 release.